docs/tfengine/schemas/project.md

```diff
@@ -6,6 +6,7 @@
 
 | Property | Description | Type | Required | Default | Pattern |
 | -------- | ----------- | ---- | -------- | ------- | ------- |
+| iam_bindings | IAM bindings for this or other projects.        See [iam_bindings.md](./iam_bindings.md) for schema. | - | false | - | - |
```

Comment:
"Addition non-authoritative IAM bindings" is probably good enough as it's not just for projects, but for buckets, billing accounts, folders, etc